components/globals/Base.js

```diff
@@ -39,7 +39,7 @@ const Base = ({ children }) => {
           </header>
         )}
         <main id="content">{children}</main>
-        {!isEmbeddable && <Footer />}
+        {!isEmbeddable && <Footer formRecord={formRecord} />}
```

Comment:
It's often best practice in React applications to only pass the needed props to a component.  In this case only the `disableGcBranding` is needed for the component to function.
```js
{!isEmbeddable && <Footer disableGcBranding=props.formRecord?.formConfig?.form?.brand?.disableGcBranding}
```